### ISSUE TITLE
Escape vertical bar in a table

### ISSUE THREAD

**Issue by tjkellyMagellanHealth** [textile/textile-lang-com-website#277]
vertical bar literal |

vertical bar in decimal &#124;

vertical bar literal in table is interpreted two columns

Sample:
|  echo "Hello" |  grep H   |

Question: Can a vertical bar be escaped in a table?

Question: There is a Unicode Symbol for "CopyRight". (C) 

Suggestion: How about a Unicode Symbol for vertical line (V) or (l)?

Revised Sample:
Sample with V:
|  echo "Hello" (V)  grep H   |
Sample with l:

|  echo "Hello" (l)  grep H   |

**Comment by jools-r** [comment]
> Question: Can a vertical bar be escaped in a table?

Yes, you can wrap it in notextile marks, e.g `== | ==` to make it appear in its own right in a table.

Try entering the following on the [tables demo page](https://textile-lang.com/doc/tables):

```
| A | simple | table | row |
| And | another  | table | row |
| With a | dual ==  | == item | -empty- | cell |
```
